docker/pypi/wmagent/init.sh

```diff
@@ -521,20 +552,20 @@ agent_resource_opp() {
 
 agent_upload_config(){
     # A function to be used for uploading WMAgentConfig to AuxDB at Central CouchDB
-    # NOTE: The final config/.dockerInit is to be set here after full agent initialisation.
+    # NOTE: The final config/.initUsing is to be set in the main() function after full agent initialization.
     #       Once we have reached to the step of uploading the agent config to Cerntral CouchDB
-    #       we consider all previous reintialisation and config steps as successfully complpeted.
+    #       we consider all previous re-internationalist and config steps as successfully completed.
```

Comment:
re-internationalist?

Reply:
hahaha  :) man this is what happens when one uses a spellchecker at  3 AM